test/data/membership_invalid.csv

```diff
@@ -5,316 +5,3 @@ neitherhere/8,northere/2
 members/8,clubs/3

```

Comment:
Can we change this filename to something like `membership_invalid_missing_header.csv` or something like that?

Reply:
This diff is a bit misleading, as it doesn't show the top of the file. The top of the file contains undefined node tables, so a good name for this might be something like `membership_invalid_referenced_tables.csv`.

Reply:
Oops!

Let's go with `membership_invalid_nonexistent_tables.csv`